src/github/branch.ts

```diff
@@ -81,11 +81,11 @@ export const deleteBranchWebhookHandler = async (context: WebhookContext, jiraCl
 	context.log.info({ prRef: context.payload.ref }, `Deleting branch for repo ${context.payload.repository?.id}`);
 
 	const jiraResponse = await jiraClient.devinfo.branch.delete(
-		await transformRepositoryId(payload.repository?.id, context.gitHubAppConfig?.gitHubBaseUrl),
```

Comment:
So yup, ALL webhook context will have a solid `gitHubAppConfig`, either for cloud or ghe app.